Index: GameSession.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/GameSession.java b/GameSession.java
--- a/GameSession.java	
+++ b/GameSession.java	
@@ -1,4 +1,4 @@
-public class gamesession {
+public class GameSession {
    
    
 //    private int moveCounter = 0;
Index: Application.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Application.java b/Application.java
--- a/Application.java	
+++ b/Application.java	
@@ -1,5 +1,5 @@
 public class Application {
     public static void main(String[] args) {
-        gamesession.printIntro();
+        GameSession.printIntro();
     }
 }
